conf/application.conf

```diff
@@ -38,23 +33,38 @@ play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-application.secret="jIk4O9ZvQkKdIgAuSYctYsdxPCTfz0wpeJ6kTBZmdcdxaHTug4gq9p4Yv2VqncPX"
+application.secret = "jIk4O9ZvQkKdIgAuSYctYsdxPCTfz0wpeJ6kTBZmdcdxaHTug4gq9p4Yv2VqncPX"
 
 # Session configuration
 # ~~~~~
-application.session.httpOnly=false
+application.session.httpOnly = false
 
-application.session.secure=false
+application.session.secure = false
 
 # The application languages
 # ~~~~~
-application.langs="en"
+application.langs = "en"
+
+
+
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule",
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
+
+
+play.modules.enabled += "uk.gov.hmrc.play.graphite.GraphiteMetricsModule"
+play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
+play.modules.enabled += "uk.gov.hmrc.servicedependencies.SchedulerModule"
+                         
+play.application.loader= "uk.gov.hmrc.play.bootstrap.HmrcApplicationLoader"
```

Comment:
Use the most recent version of bootstrap (0.9.0), where this has been renamed.

Reply:
Upgraded to bootstrap 0.10.0 and changed the loader to uk.gov.hmrc.play.bootstrap.ApplicationLoader